gulp/tasks/styles.js

```diff
@@ -1,34 +1,45 @@
-var gulp = require('gulp'),
-  config = require('../config'),
-  gutil = require('gulp-util'),
-  stylus = require('gulp-stylus'),
-  environments = require('gulp-environments'),
-  concat = require('gulp-concat');
-
-var development = environments.development;
-var production = environments.production;
-
-
-gulp.task('stylus', ['clean'], function(){
-  var compress = production()
-  return gulp.src(config.paths.css.src)
-    .pipe(stylus({
-      compress: production()
-    }))
-    .pipe(gulp.dest(config.paths.temp))
-});
-
-var tasks = ['stylus'];
-for (var key in config.paths.css.bundles){
-  (function(key) {
-    tasks.push('styles:'+key);
-    gulp.task('styles:'+key, ['clean', 'stylus'], function() {
-      var dest = production() ? config.paths.css.dest.production : config.paths.css.dest.development;
-      return gulp.src(config.paths.css.bundles[key])
-        .pipe(concat(key+'.css'))
-        .pipe(gulp.dest(dest))
-    });
-  })(key);
+"use strict";
+
+const { src, dest, series } = require("gulp");
+const stylus = require("gulp-stylus");
+const { production } = require("gulp-environments");
+const concat = require("gulp-concat");
+
+const config = require("../config");
+
+const stylusTask = () => {
```

Comment:
no material changes here, just used a newer form of task declaration (namely, `task(...)` is deprecated)